Week2/homework/map-filter.js

```diff
@@ -1,8 +1,7 @@
 'use strict';
 
 function doubleOddNumbers(numbers) {
-  // Replace this comment and the next line with your code
-  console.log(numbers);
+  return numbers.filter(elem => (elem % 2 !== 0 ? elem : false)).map(elem => elem * 2);
```

Comment:
Same here, take one more look at the docs to refresh your memory.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter